### PR TITLE
Runtime hunting: heater wrenching and rotate verbs

### PR DESCRIPTION
**Pull request by DamianX**
```
Runtime in Freezer.dm,258: Cannot execute null.disconnect().
  proc name: wrenchAnchor (/obj/machinery/atmospherics/unary/heat_reservoir/heater/wrenchAnchor)
```

### CODE REVIEW

File: code/game/machinery/Freezer.dm [hunk 1]
```diff
@@ -253,12 +254,14 @@
 		if (node)
 			node.initialize()
 			node.build_network()
-		verbs += rotate_verbs
 	else
-		node.disconnect(src)
-		qdel(network)
-		network = null
-		node = null
+		verbs += rotate_verbs
```
Comment (gbasood) [inline code/game/machinery/Freezer.dm:258]: Should this line have moved beneath the `else`?

Comment (DamianX) [reply code/game/machinery/Freezer.dm:258]: yeah

Comment (gbasood) [reply code/game/machinery/Freezer.dm:258]: Oh, I misread the conditions the first time through, neat

**Review by gbasood — APPROVED** [review]

**Comment by gbasood** [comment]
looks like the build has passed but github hasn't picked up on that yet

**Pull request merged**